src/main/java/com/ExamHub/ExamHub/controller/examController/ExamController.java

```diff
@@ -0,0 +1,98 @@
+            if (savedExam != null) {
+                return new ResponseEntity<Message>(new Message(HttpStatus.OK, MessageConstant.EXAM_SAVED, CommonConstants.SUCCESS), HttpStatus.OK);
+            } else {
+                return new ResponseEntity<Message>(new Message(HttpStatus.BAD_REQUEST, CommonConstants.EXCEPTION, CommonConstants.EXCEPTION), HttpStatus.BAD_REQUEST);
+            }
+        } catch (Exception e) {
+            return new ResponseEntity<Message>(new Message(HttpStatus.BAD_REQUEST, e.getMessage(), CommonConstants.EXCEPTION), HttpStatus.BAD_REQUEST);
+        }
+    }
+
+
+   // @ApiOperation(value = "update Exam")
+    @RequestMapping(value = RequestMappingConstants.UPDATEEXAM, consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE, method = RequestMethod.POST)
+    public ResponseEntity<Message> updateExam(@RequestBody ExamRequestDto examRequestDto) throws Exception {
+
+        try {
+
+            Exam savedExam = examService.updateExam(examRequestDto);
+            if (savedExam != null) {
+                return new ResponseEntity<Message>(new Message(HttpStatus.OK, MessageConstant.EXAM_UPDATE, CommonConstants.SUCCESS), HttpStatus.OK);
+            } else {
+                return new ResponseEntity<Message>(new Message(HttpStatus.BAD_REQUEST, CommonConstants.EXCEPTION, CommonConstants.EXCEPTION), HttpStatus.BAD_REQUEST);
+            }
+        } catch (Exception e) {
+            return new ResponseEntity<Message>(new Message(HttpStatus.BAD_REQUEST, e.getMessage(), CommonConstants.EXCEPTION), HttpStatus.BAD_REQUEST);
+        }
+    }
+
+  //  @ApiOperation(value = "get all Exam")
+    @RequestMapping(value = RequestMappingConstants.GETALLEXAM, params = {"page", "size"}, method = RequestMethod.GET)
+    public ResponseEntity<?> getAllExam(@RequestParam("page") int page, @RequestParam("size") int size) throws InstanceNotFoundException {
+
+        if (size == 0) {
+            throw new IncorrectResultSizeDataAccessException("Expected PageSize not Found.", 10, size);
+        }
+        Page<Exam> result = examService.getAllExam(page, size);
+        if (CollectionUtils.isEmpty(result.getContent())) {
+            return new ResponseEntity<>(new ResultList<Exam>(result.getContent(), new Message(HttpStatus.OK, MessageConstant.NOT_FOUND, CommonConstants.SUCCESS)), HttpStatus.OK);
+        }
+        return ResponseEntity.ok(new ResultList<Exam>(result.getContent(), result.getPageable(), result.getTotalPages(), result.getTotalElements(), new Message(HttpStatus.OK, CommonConstants.SUCCESS, CommonConstants.SUCCESS)));
+    }
+
+   // @ApiOperation(value = "get Exam by id")
+    @RequestMapping(value = RequestMappingConstants.GETEXAMBYID, params = {"examId"}, method = RequestMethod.GET)
+    public ResponseEntity<?> getExamById(@RequestParam("examId") int examId)  {
+        Exam exam = examService.getExamById(examId);
+        return ResponseEntity.ok(exam);
```

Comment:
Add null check here and then return the response, if the exam is empty then return error message